### PR TITLE
fix accidental removal of Qemu Disk storage_type and format

### PR DESCRIPTION
**Pull request by Matchlighter**
See https://github.com/Telmate/proxmox-api-go/pull/96#commitcomment-44008540

### CODE REVIEW

**Comment by Matchlighter** [comment]
@yukron Does this look like a sufficient fix?

**Comment by yukron** [comment]
> @yukron Does this look like a sufficient fix?

Yes, this should work.

File: proxmox/config_qemu.go [hunk 1]
```diff
@@ -595,6 +595,32 @@ func NewConfigQemuFromApi(vmr *VmRef, client *Client) (config *ConfigQemu, err e
...
+		storageContent, err := client.GetStorageContent(vmr, storageName)
+		if err != nil {
+			log.Fatal(err)
+			return nil, err
+		}
+		var storageFormat string
+		contents := storageContent["data"].([]interface{})
+		for content := range contents {
+			storageContentMap := contents[content].(map[string]interface{})
+			if storageContentMap["volid"] == filePath {
```
Comment (mleone87) [inline proxmox/config_qemu.go:607]: @Matchlighter **filePath** does not exist anywhere in the code

Comment (yukron) [reply proxmox/config_qemu.go:607]: @mleone87 I've included a fix for this in my PR #98

Comment (Matchlighter) [reply proxmox/config_qemu.go:607]: Thanks @yukron. I must have built in the wrong dir/branch - I feel so foolish...

**Pull request merged**